### ISSUE TITLE
Stop robot from buffering driver inputs while disabled

### ISSUE THREAD

**Issue by ewpratten** [frc5024/DeepSpace-SWI#2]
## Problem
While disabled, if the driver / operator presses a button, the action will be completed when the bot is re-enabled. This can lead to dangerous situations (think, starting climb while disabled on a workbench, then enabling to test something else)

### Steps to reproduce
Steps to reproduce the behaviour:
1. Turn on robot (or reboot)
2. Ensure robot is in a safe place **away from people**
3. Disable robot from DriverStation
4. Press the `Y` button on the *operator's controller* (This normally places a hatch)
5. Enable robot soon after step 4

The robot will then execute a hatch placement once the robot is enabled.

### Expected behaviour
All human inputs while the robot is disabled should be ignored, and all request states should be reset when the bot is enabled. If a system requires different behaviour, a work-around can be implemented as needed.

## Proposed solution
A method should be added to the [SubsystemLooper](https://github.com/frc5024/DeepSpace-SWI/blob/1d56b4379f1b51c55a8d503635b70c38e1f0b89a/src/main/java/frc/lib5k/loops/loopers/SubsystemLooper.java) that calls the `stop()` method of every registered `LoopableSubsystem`. This method will then be called by `Robot`'s `teleopInit()` and `autonomousInit()` methods. This will ensure that any action requests or inputs are cleared when the robot is enabled.

### Execution
A method named `stopAll()` is to be added to the `SubsystemLooper` class.

`stopAll()` will iterate through every registered `LoopableSubsystem` and call that subsystem's `stop()` method. An iterable list of registered subsystems already exists, and is called `subsystems` inside the `LoopableSubsystem` class.

### Testing
For this issue to be considered resolved, you must demonstrate 5 times that the issue can not be reproduced. If this test is successful, the code will be merged into testing, then master.

**Comment by ewpratten** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Fix subsystem buffering issue](https://trello.com/c/VkWlW9fJ/231-fix-subsystem-buffering-issue)

**Comment by ewpratten** [comment]
fixed by #4
